Bottom: d1e910f0bcfd70e42f171ac7e5f10fc2d7ec4d5e
Top:    11733402ecb6e8f19b2ca2972cd200304643c8c6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 17:46:04 +0000

Refresh of make-hid_draw-an-instance-of

---

diff --git a/src/hid/common/hidnogui.c b/src/hid/common/hidnogui.c
index 64dd8d5..3b85044 100644
--- a/src/hid/common/hidnogui.c
+++ b/src/hid/common/hidnogui.c
@@ -525,10 +525,11 @@ hid_nogui_get_hid (void)
                           "Does nothing.";
   nogui_hid.graphics    = &nogui_graphics;
 
-  nogui_graphics.klass  = &nogui_graphics_class;
-
   common_nogui_init (&nogui_hid);
+
   common_nogui_graphics_class_init (&nogui_graphics_class);
+
+  nogui_graphics.klass = &nogui_graphics_class;
   common_nogui_graphics_init (&nogui_graphics);
 
   return &nogui_hid;
diff --git a/src/hid/nelma/nelma.c b/src/hid/nelma/nelma.c
index df12c26..cb28849 100644
--- a/src/hid/nelma/nelma.c
+++ b/src/hid/nelma/nelma.c
@@ -1098,7 +1098,7 @@ hid_nelma_init()
   nelma_graphics_class.fill_polygon   = nelma_fill_polygon;
   nelma_graphics_class.fill_rect      = nelma_fill_rect;
 
-  nelma_graphics.klass          = &nelma_graphics_class;
+  nelma_graphics.klass = &nelma_graphics_class;
   common_draw_helpers_init (&nelma_graphics);
 
   hid_register_hid (&nelma_hid);
diff --git a/src/hid/ps/eps.c b/src/hid/ps/eps.c
index 5e56e70..5890e4c 100644
--- a/src/hid/ps/eps.c
+++ b/src/hid/ps/eps.c
@@ -713,7 +713,7 @@ hid_eps_init ()
   eps_graphics_class.fill_polygon   = eps_fill_polygon;
   eps_graphics_class.fill_rect      = eps_fill_rect;
 
-  eps_graphics.klass          = &eps_graphics_class;
+  eps_graphics.klass = &eps_graphics_class;
   common_draw_helpers_init (&eps_graphics);
 
   hid_register_hid (&eps_hid);
diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 5c46206..a52bf76 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -1577,7 +1577,7 @@ hid_ps_init ()
   common_draw_helpers_class_init (&ps_graphics_class);
   ps_ps_graphics_class_init (&ps_graphics_class);
 
-  ps_graphics.klass         = &ps_graphics_class;
+  ps_graphics.klass = &ps_graphics_class;
   common_draw_helpers_init (&ps_graphics);
   ps_ps_graphics_init (&ps_graphics);
